### ISSUE TITLE
telnet escape character

### ISSUE THREAD

**Issue by dpassen** [akermu/emacs-libvterm#368]
When using vterm to telnet, I am unable to type the escape character "^]"
```sh
$ telnet www.google.com 80                   
Trying 216.58.192.228...
Connected to www.google.com.
Escape character is '^]'.
```

**Comment by Sbozzolo** [comment]
You can add the key with this:
```(define-key vterm-mode-map (kbd "C-]") #'vterm--self-insert)```

I'll fix this in a future PR.

**Comment by dpassen** [comment]
Thanks!

**Comment by jixiuf** [comment]
closed by 9469bdf461dfe33057e049d611e12a8e533054de
